sda-commons-starter/src/main/java/org/sdase/commons/starter/builder/SecurityCustomizer.java

```diff
@@ -12,4 +12,14 @@
    * @return this builder instance
    */
   PlatformBundleBuilder<C> disableBufferLimitValidationSecurityFeature();
+
+  /**
+   * If a service is configured with frontend support, the {@code Content-Security-Policy} header
+   * allows the same domain as source for scripts, images, styles and fonts. Otherwise, only API
+   * endpoints can be served and {@code Content-Security-Policy} does not allow any sources.
+   *
+   * @return this builder instance
+   * @see <a href="https://en.wikipedia.org/wiki/Content_Security_Policy">CSP header</a>
+   */
+  PlatformBundleBuilder<C> withFrontendSupport();
```

Comment:
It would not be possible to use both options now, would it? Both methods return `PlatformBundleBuilder` and not `SecurityCustomizer`.

Reply:
[`PlatformBundleBuilder`](https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-starter/src/main/java/org/sdase/commons/starter/builder/PlatformBundleBuilder.java#L13) extends `SecurityCustomizer`.